matchbox_server/src/signaling.rs

```diff
@@ -452,9 +459,212 @@ mod tests {
             _ = &mut timeout => {}
         }
     }
+    #[tokio::test]
+    async fn match_pair_and_other_alone_room_without_next() {
```

Comment:
Here we check:

- player a -> scope_1?next=2
- player b -> scope_1
- player c -> scope_1?next=2